docs-ref-conceptual/azure-cli-endpoints.md

```diff
@@ -0,0 +1,220 @@
+vm_image_alias_doc | https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/arm-compute/quickstart-templates/aliases.json
+media_resource_id | https://rest.media.cloudapi.de
+ossrdbms_resource_id | https://ossrdbms-aad.database.cloudapi.de
+portal | https://portal.microsoftazure.de
+
+### Endpoint suffixes
+
+|Suffix name | Suffix
+|-|-|
+storage_endpoint | *.core.cloudapi.de
+keyvault_dns | *.vault.microsoftazure.de
+mhsm_dns | *.managedhsm.microsoftazure.de
+sql_server_hostname | *.database.cloudapi.de
+mysql_server_endpoint | *.mysql.database.cloudapi.de
+postgresql_server_endpoint | *.postgres.database.cloudapi.de
+mariadb_server_endpoint | *.mariadb.database.cloudapi.de
+
+---
+
+## Extension endpoints
+
+Azure CLI extensions are optional and installed separately.  All extensions install with endpoint **azcliprod.blob.core.windows.net** with the following exceptions:
+
+|Extension name | Endpoint
+|-|-|
+| arcappliance | arcplatformcliextprod.blob.core.windows.net
+| customlocation |
+| arcdata | azurearcdatacli.blob.core.windows.net
+| azure-batch-cli-extensions | github.com\Azure
+| azure-devops |
+| azure-iot |
+| baremetal-infrastructure |
+| csvmware |
+| deploy-to-azure |
+| sap-hana |
+| azure-cli-ml | azurecliext.blob.core.windows.net
+| azurestackhci | hybridaksstorage.z13.web.core.windows.net
+| Hybridaks |
+| fzf | pahealyfzf.blob.core.windows.net
+|image-copy-extension | files.pythonhosted.org
+| managementpartner |
+| resource-graph |
+| ml | azuremlsdktestpypi.blob.core.windows.net
+| qbs | qbsazcliextension.blob.core.windows.net
+
+To see a list of installed extension for the Azure CLI, enter `az` at your terminal command prompt and press **enter**.  To see a list of available extensions see [Available Azure CLI extensions](/cli/azure/azure-cli-extensions-list).
+
+## Engineering questions
+1. What about **pypi.org** and **files.pythonhosted.org**?  According to the customer request for installing the `connectedk8s` extension, they had to allow these two URLs.  I only see **files.pythonhosted.org** under `image-copy-extension`. **pypi.org** isn't in any list.
+
+## See also
+
+* [How to use the Azure CLI effectively - Work behind a proxy](/cli/azure/use-cli-effectively&tabs=bash%2Cbash2#work-behind-a-proxy)
+* [Work with existing on-premises proxy servers](/azure/active-directory/app-proxy/application-proxy-configure-connectors-with-proxy-servers)
+* [Azure Firewall threat intelligence configuration - Allowlist addresses](/azure/firewall-manager/threat-intelligence-settings#allowlist-addresses)
+* Azure IP ranges and Service Tags
+  [Public Cloud](https://www.microsoft.com/en-us/download/details.aspx?id=56519)
+  [US Government Cloud](https://www.microsoft.com/en-us/download/details.aspx?id=57063)
+  [Germany Cloud](https://www.microsoft.com/en-us/download/details.aspx?id=57064)
+  [China Cloud](https://www.microsoft.com/en-us/download/details.aspx?id=57062)
+
+omit these related links
```

Comment:
Omit these related links?

Reply:
@chasewilson , yes, I just left them there in the event you thought I should include them.  They are related, but there are 70+ docs that talk about allowlists and firewalls.  Which related content is applicable here?